src/land/UberArcanistSubmitQueueEngine.php

```diff
@@ -91,6 +98,89 @@ private function identifyRevision() {
     }
   }
 
+  /**
+   * Validate if autoland tag should be added to the revision and actual landing
+   * procedure should be stopped. Return false if tagging was skipped.
+   */
+  private function tagWithAutoland($revision) {
+    // Check if explicit flags are provided
+    if (true === $this->getTbr()) {
+      return false;
+    }
+
+    // check if we have TTY otherwise it is probably automation...
+    try {
```

Comment:
```
if ($this->getArgument('nointeractive')) {
      return false;
    }
```
noninteractive is also an option used by some mobile/java automations...

Reply:
or was this on purpose removed?

Reply:
`--noninteractive` flag is available on `arc diff` workflow only but not `arc land`.